<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkPageMore</class>
 <widget class="QWidget" name="NetworkPageMore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <widget class="QLabel" name="label_pushStream">
       <property name="text">
        <string>Push Video Stream</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_pushMsg">
       <property name="text">
        <string>Push Message</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_ssh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>SSH Enable</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ComboBox" name="comboBox_pushStream">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_sshPort">
       <property name="text">
        <string>SSH Port</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_pushType">
       <property name="text">
        <string>Push Message Settings</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="MyLineEdit" name="lineEdit_sdkPort">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="MyLineEdit" name="lineEdit_httpsPort">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="comboBox_ssh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_httpsPort">
       <property name="text">
        <string>HTTPS Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MyLineEdit" name="lineEdit_sshPort">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="MyLineEdit" name="lineEdit_rtspPort">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_sdkport">
       <property name="text">
        <string>Service Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="MyLineEdit" name="lineEdit_httpPort">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ComboBox" name="comboBox_pushMsg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_httpPort">
       <property name="text">
        <string>HTTP Port</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_rtspPort">
       <property name="text">
        <string>RTSP Port</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyPushButton" name="pushButton_pushType">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>82</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
